recipes/dnp-binstrings/meta.yaml

```diff
@@ -0,0 +1,33 @@
+package:
+  name: binstrings
+  version: "1.0"
+
+source:
+  url: https://github.com/erinijapranckeviciene/dnpatterntools/files/2431226/v1.0.tar.gz
+  sha256: bb38cf23132445d9c2ffec04229a578207eb932c38a5cfc8168d5c9a79f94a39
+
+build:
+  number: 1
```

Comment:
Please do start new version of a recipe always with 0.

Reply:
Thank you, yes, will start with 0 in all.